src/H5PLpkg.h

```diff
@@ -153,7 +154,8 @@ H5_DLL herr_t      H5PL__replace_path(const char *path, unsigned int index);
 H5_DLL herr_t      H5PL__insert_path(const char *path, unsigned int index);
 H5_DLL herr_t      H5PL__remove_path(unsigned int index);
 H5_DLL const char *H5PL__get_path(unsigned int index);
-H5_DLL herr_t      H5PL__find_plugin_in_path_table(const H5PL_search_params_t *search_params,
-                                                   hbool_t *found /*out*/, const void **plugin_info /*out*/);
+H5_DLL herr_t H5PL__path_table_iterate(H5PL_iterate_type_t iter_type, H5PL_iterate_t iter_op, void *op_data);
+H5_DLL herr_t H5PL__find_plugin_in_path_table(const H5PL_search_params_t *search_params,
```

Comment:
Clang seems to be confused about the formatting here, but this is what it wants, so it's what goes.